src/Builder/Type/OperatorInterface.php

```diff
@@ -9,8 +9,11 @@
  */
 interface OperatorInterface
 {
-    /** To be overridden by implementing classes */
+    /** @var Encode */
     public const ENCODE = Encode::Undefined;
```

Comment:
I was wondering why we didn't use a typed class constant, until I saw that they were only [added in PHP 8.3](https://wiki.php.net/rfc/typed_class_constants).